WalletWasabi.Backend/InitConfigStartupTask.cs

```diff
@@ -57,13 +58,11 @@ public async Task ExecuteAsync(CancellationToken cancellationToken)
 					network: config.Network);
 
 			var cachedRpc = new CachedRpcClient(rpc, Cache);
-
-			var pushService = new SendPushService(ContextFactory, config);
-			await Global.InitializeAsync(config, roundConfig, cachedRpc, pushService, cancellationToken);
+			await Global.InitializeAsync(config, roundConfig, cachedRpc, _serviceProvider, cancellationToken);
 
 			try
 			{
-				await WebsiteTorifier.CloneAndUpdateOnionIndexHtmlAsync();
+				await _websiteTorifier.CloneAndUpdateOnionIndexHtmlAsync();
```

Comment:
Wasabi has code cleanup rules that yell at you for this private readonly naming convention